solid_attenuator/ioc_lfe_at2l0_calc/calculator.py

```diff
@@ -17,6 +17,7 @@
 import itertools
 import pathlib
 import typing
+from typing import Tuple
```

Comment:
Stylistically, how do you pick when to use `import typing` vs `from typing import ...`?

Reply:
It's usually `typing.X` until I start using `X` a few times - or there's a scenario where `typing.X` doesn't fit on one line but `X` does...